geo_map/static/geo_map/multiselect.js

```diff
@@ -184,7 +198,7 @@ function MultiselectDropdown(options) {
         if (0 == el.selectedOptions.length)
           div.appendChild(
             newEl("span", {
-              class: "placeholder",
+              class: ["placeholder", "ts-control", "bg-surface-secondary"],
```

Comment:
the `placeholder` class name collides with netbox's internal classes. consider removing all classes and replacing them with a unique one that will just hide the placeholder if there are selected items